PC02_Graffiti_MCNULTY.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
Overall, the quality of this code is moderate. It runs effectively without flaws, but it could be more concise and also could have utilized more advanced commands. Additionally, using a more descriptive turtle name is something that should have been taken into consideration. In total, the program drew a funny image of a crown, ice cream cone, and an earring and also showed an understanding of the begin_fill() and end_fill() functions.